command/commands/team.py

```diff
@@ -102,6 +110,17 @@ def init_subparsers(self):
                                  help="Display name the team should have.")
         parser_edit.add_argument("--platform", type=str, action='store',
                                  help="Platform the team should have.")
+
+        """Parser for lead command."""
+        parser_lead = subparsers.add_parser("lead")
+        parser_lead.set_defaults(which='lead',
+                                 help="Edit leads of specified team.")
+        parser_lead.add_argument("team_name", type=str, action='store',
```

Comment:
I believe we're changing all these underscores to hyphens (@cheukyin699?).

Reply:
oh boy

Reply:
so I tried replacing `team_name` with `team-name` but I get an error whenever I call `args.team-name` where it says `name` is an unresolved reference

Reply:
no idea if this helps but https://stackoverflow.com/questions/12834785/having-options-in-argparse-with-a-dash

Reply:
Should i just change the `team_name` in the add_argument lines and none of the others?

Reply:
you should keep that as `team_name` and instead add parameter `metavar='team-name'`. That way, you can keep using `arg.team_name` while also displaying `team-name`.

Reply:
Above was taken from link that @ravenCondol found.